pkgs/by-name/ph/photoprism/backend.nix

```diff
@@ -2,14 +2,24 @@
   lib,
   buildGoModule,
   coreutils,
-  libtensorflow,
+  python3,
   src,
   version,
   pkg-config,
   vips,
+  runCommand,
 }:
 
-buildGoModule rec {
+let
+  # we need to copy these, to add the symlinks, so the linker actually finds these libraries
+  libtensorflow = runCommand "libtensorflow" { } ''
```

Comment:
is it possible to express this as a link farm? Perhaps using `symlinkJoin` with a `postBuild`?